pages/solid-events.md

```diff
@@ -14,7 +14,7 @@ Are you interested in speaking about what you have been building on Solid? Reach
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |----|-------|-------------|------------------------------|
-| 2021-01-07 | [Solid World January](https://www.eventbrite.com/e/solid-world-tickets-132301957717)|[Solid Team](https://solidproject.org/team)||
+| 2021-01-07 | [Solid World January](https://www.eventbrite.com/e/solid-world-tickets-132301957717)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/498076263)|
```

Comment:
Just wondering if we should start a new table for 2021 events?

Reply:
Sounds good to me - Marrelle was also suggesting some changes to the layout here, so maybe we can do that as part of that: https://github.com/solid/solidproject.org/pull/484

(Also: didn't notice you were a Creator now too, that's great :) )